src/components/NavBar.tsx

```diff
@@ -17,9 +17,26 @@ function NavBar() {
       (item === 'Contact Us' && pageName === 'Contact')
```

Comment:
consider abstracting, and is there a simpler way to check for true value?

Reply:
fixed in 070b0ec
but nt sure whether this is what you want

Reply:
actl I was trying to ask you if there is a simpler/shorter way to check for true value cos it seems a little verbose to be haha; if not then it's fine.

also right the `item.split(' ')[0] === pageName` seems arbitrary and hard-to-understand (unless you're sure that the pageName is always the first word in item), so would much prefer the previous version

Reply:
oh ok LOL